reanahub/reana-server
# fetcher: generate workflow name from given URL

Closes #449

How to test:
Launch some workflows from `/launch?url=...` and check the name of the created workflow:
```
https://github.com/reanahub/reana-demo-root6-roofit
https://zenodo.org/record/5752285/files/circular-health-data-processing-master.zip?download=1
https://raw.githubusercontent.com/reanahub/reana-demo-root6-roofit/master/reana.yaml
```
